### DISCUSSION TITLE
issue to setup in local

### DISCUSSION THREAD

**Discussion by MdTanwer** [resend/react-email#1581] [General]
i try to set up in my local mashine but show this error any body help me

![image](https://github.com/user-attachments/assets/029c1083-8d8e-4b4e-8300-72d7d324d781)

**Comment by gabrielmfern** [comment]
We need to improve our contributing guide on this, but what you should do is:
- `pnpm build --filter=./packages/*`
- `pnpm install`
- `pnpm build`

`pnpm dev` isn't really necessary here, we never it ourselves, might be removed later on. You should just run `pnpm dev` on the package/app that you are going to be developing on.
